website/src/translation/en-US.tsx

```diff
@@ -0,0 +1,198 @@
+import type { TooltipProps } from "../components/ui/Tooltip";
+import type { TranslationComponent, TranslationObject } from "./";
+
+export const translation: TranslationObject = {
+	htmlLang: () => "en",
+	documentationTitle: () => "Typst Documentation (English Version)",
+	close: () => "Close",
+	closeMenu: () => "Close menu",
+	closeSearch: () => "Close search",
+	openMenu: () => "Open menu",
+	openSearch: () => "Open search",
+	showInformation: (props: { name: string }) =>
+		`Show details for ${props.name}`,
+	tooltipKind: (props: { kind: TooltipProps["kind"] }) => {
+		switch (props.kind) {
+			case "element":
+				return "Element function";
+			case "contextual":
+				return "Context function";
+			case "definitions":
+				return "Definition";
+			case "parameters":
+				return "Parameter";
+			case "variadic":
+				return "Variadic parameter";
+			case "settable":
+				return "Settable parameter";
+			case "positional":
+				return "Positional parameter";
+			case "required":
+				return "Required parameter";
+			default:
+				return props.kind;
+		}
+	},
+} as const;
+
+export const Translation: TranslationComponent = (props) => {
+	switch (props.translationKey) {
+		case "definition":
+			return <>Definition</>;
+		case "constructor":
+			return <>Constructor</>;
+		case "definitionOf":
+			return (
+				<>
+					<code>{props.name}</code> Definition
+				</>
+			);
+		case "search":
+			return <>Search</>;
+		case "defaultValue":
+			return <>Default value:</>;
+		case "stringValues":
+			return <>Available string values:</>;
+		case "showExample":
+			return <>Show example</>;
+		case "tableOfContents":
+			return <>Table of contents</>;
```

Comment:
```suggestion
			return <>On this page</>;
```

Otherwise, it might be confused with https://typst.app/docs/reference/model/outline/.